### ISSUE TITLE
which recipe to suggest for librispeech 

### ISSUE THREAD

**Issue by riyijiye** [vrenkens/nabu#30]
Hi Vincent,

I want to try running an LAS experiment with your code using a subset of librispeech data (100 hours), I am wondering which recipe I should start with?  looks to me this is close to WSJ  (50 hour and also read speech), should I use this recipe config/recipes/LAS/WSJ? or what do you suggest? 

thank you!
Harry

**Comment by riyijiye** [comment]
by the way, forgot to mention, I plan to start an experiment for character-based model

**Comment by riyijiye** [comment]
by the way, how many epoches you really played, I saw in timit and wsj recipe 100, 300 etc. it is really big numbers

**Comment by riyijiye** [comment]
by the way, what is the reason of using sum of ce-eos loss instead of averaging? to me using averaging  is easier to monitor training batch loss change across training steps since it is length-normalized (validation loss is ok since validation set is always fixed)

**Comment by vrenkens** [comment]
Hi Harry,

The WSJ probably makes sense. As for epochs I think it is set so high because I wanted to use the dev set to monitor training and use early stopping. I agree that it is probably way to many epochs :). As for the Loss. If you average the loss instead of summing you give a lower importance to the longer utterances because you divide their loss by a higher number and I did not feel this was appropriate behavior :).

**Comment by riyijiye** [comment]
thanks Vincent!
